src/convertToObject.js

```diff
@@ -7,6 +7,20 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString
+    .trim()
+    .split(';')
+    .map((style) => style.trim())
+    .filter((style) => style)
+    .reduce((stylesObject, style) => {
+      const [property, value] = style.split(':').map((part) => part.trim());
+
+      if (property && value) {
+        stylesObject[property] = value;
```

Comment:
The check for both `property` and `value` being truthy before adding them to the stylesObject is a good practice to avoid adding invalid entries.